riscv/src/code_gen.rs

```diff
@@ -315,6 +315,8 @@ let initial_memory: (fe, fe)[] = [
 }}    
 "#,
         runtime.submachines_import(),
+        1 << (powdr_linker::MIN_DEGREE_LOG - 2),
+        1 << (*powdr_linker::MAX_DEGREE_LOG - 2),
```

Comment:
There is a bad reason: if we set min and max to the same value, the linker assumes that we are in the monolithic case and sets all degrees to that same value..

Reply:
Oh, but I didn't suggest setting it to the same value. Not quite sure about the tradeoffs, but I think it could be a useful feature that we can prove the last chunk (or only chunk in case of a small computation) faster because it is smaller. But going below `1 << MIN_DEGREE_LOG` (=32 in practice) seems unnecessary, proving 32 rows should take no time.

Reply:
Oh I misread your suggestion! Yes I agree.